sdk/test/ProtocolTests/AWSSDK.ProtocolTests.Netstandard.csproj

```diff
@@ -0,0 +1,63 @@
+      <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+      <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+      <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+      <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+      <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+      <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+      <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+      <SignAssembly>true</SignAssembly>
+  
+      <NoWarn>CS1591</NoWarn>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Compile Include="./Generated/**" />
+        <Compile Include="./Custom/**" />
+    </ItemGroup>
+    <ItemGroup>
+      <Compile Remove="**/obj/**" />
+      <None Remove="**/obj/**" />
+    </ItemGroup>
+  
+    <ItemGroup>
+      <ProjectReference Include="../../src/Core/AWSSDK.Core.Netstandard.csproj" />
+	  <ProjectReference Include="../Netstandard//Common/AWSSDK.CommonTest.Netstandard.csproj" />
+      <ProjectReference Include="../../../generator/ServiceClientGeneratorLib/ServiceClientGeneratorLib.csproj" />
+      <ProjectReference Include="../../../extensions/src/AWSSDK.Extensions.CrtIntegration/AWSSDK.Extensions.CrtIntegration.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/RestJsonProtocol/AWSSDK.RestJsonProtocol.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/JSONRPC10/AWSSDK.JSONRPC10.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/JsonProtocol/AWSSDK.JsonProtocol.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/QueryProtocol/AWSSDK.QueryProtocol.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/EC2Protocol/AWSSDK.EC2Protocol.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/RestXmlProtocol/AWSSDK.RestXmlProtocol.Netstandard.csproj" />
+	  <ProjectReference Include="../Services/RestXmlProtocolNamespace/AWSSDK.RestXmlProtocolNamespace.Netstandard.csproj" />
+	  <ProjectReference Include="../UnitTests/Custom/AWSSDK.UnitTestUtilities.Netstandard.csproj" />
+    </ItemGroup>
+    <ItemGroup Condition="$(RunAnalyzersDuringBuild)">
+    </ItemGroup>
+    <ItemGroup>
+          <PackageReference Include="Moq" Version="4.8.3" />
+          <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
+          <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+          <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
+          <PackageReference Include="AutoFixture" Version="3.51.0" />
+          <PackageReference Include="AutoFixture.AutoMoq" Version="3.51.0" />
+          <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    </ItemGroup>
+  
+    <ItemGroup>
+      <Reference Include="System.Configuration" />
```

Comment:
Is this required? I only see this `Reference` entry in `Net35` / `Net45` projects.